networkRequests/roles.js

```diff
@@ -0,0 +1,41 @@
+import API_URL from "../components/globalApiUrl";
+
+export const roleChange = async (userID, role) => {
```

Comment:
This should actually go in the "utils/api.js" folder with all the other API requests, just to keep it consistent with the rest of the repo.